### PR TITLE
README update and some cleanups.

### PR DESCRIPTION
**Pull request by fujitatomoya**

### CODE REVIEW

File: rcl_logging_noop/CMakeLists.txt [hunk 1]
```diff
@@ -21,7 +21,7 @@ if(NOT WIN32)
 endif()
 
 set(${PROJECT_NAME}_sources
-  src/rcl_logging_noop/rcl_logging_noop.cpp
+  src/rcl_logging_noop.cpp
```
Comment (fujitatomoya) [inline rcl_logging_noop/CMakeLists.txt:24]: this aligns with rcl_logging_spdlog directory tree.

File: rcl_logging_spdlog/README.md [hunk 2]
```diff
@@ -2,16 +2,12 @@
...
-[rcl_logging_spdlog](include/rcl_logging_spdlog/logging_interface.h) logging interface implementation can:
+[rcl_logging_spdlog](src/rcl_logging_spdlog.cpp) logging interface implementation can:
  - initialize
  - log a message
  - set the logger level
  - shutdown
 
-Some useful internal abstractions and utilities:
-  - Macros for controlling symbol visibility on the library
-    - [rcl_logging_spdlog/visibility_control.h](include/rcl_logging_spdlog/visibility_control.h)
```
Comment (fujitatomoya) [inline rcl_logging_spdlog/README.md:13]: this file does not exist.

**Review by fujitatomoya — COMMENTED** [review]
@clalancette @wjwwood minor update mostly docs, can you take a look?

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rcl_logging_noop/CMakeLists.txt (hunk 1 above)
Comment (fujitatomoya) [reply rcl_logging_noop/CMakeLists.txt]: https://github.com/ros2/rcl_logging/pull/120/commits/7df68baf786810a7d57d4436bc5a37f60a296468 fixes.

**Comment by fujitatomoya** [comment]
@clalancette requesting another review on this.

**Comment by fujitatomoya** [comment]
Pulls: ros2/rcl_logging#120
Gist: https://gist.githubusercontent.com/fujitatomoya/38f6efeffacfd7206e46612c78a45217/raw/276f1d6e84baef70790959dc1b765419f16697f0/ros2.repos
BUILD args:  --packages-up-to rcl_logging_noop rcl_logging_spdlog
TEST args:  --packages-select rcl_logging_noop rcl_logging_spdlog
ROS Distro: rolling
Job: ci_launcher
ci_launcher ran: https://ci.ros2.org/job/ci_launcher/14512
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21787/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/16059/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/1290/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22486/)

**Review by clalancette — APPROVED** [review]
Looks good to me with green CI.

**Review by ahcorde — APPROVED** [review]

**Pull request merged**